ovos_utils/messagebus.py

```diff
@@ -12,13 +12,18 @@
 from ovos_utils.log import LOG
 from ovos_utils.metrics import Stopwatch
 
+# TODO: Consider moving all of this to `ovos_bus_client`
```

Comment:
this is all in ovos-bus-client, but is an optional dependency of ovos-utils so it contains these wrappers

some of the methods can not be removed from here as they are used internally in the apis ovos-utils provides, the Message object from this module needs to be importable, we just dont want other packages to use this module  (except unittests and such if ovos-bus-client is also optional)

Reply:
If they're duplicated, why not just import from ovos-bus-client and otherwise raise an exception to install that extra? Otherwise we end up maintaining the same utils in 2 locations

Reply:
not duplicated code, these are just wrappers calling the optional code if the lib is available and returning default values otherwise....

the FakeMessage is duplicated but that's a companion for FakeBus

just importing from bus client causes a lot of issues on neon and docker that why those were added back, initially i just left the methods actually used internally here...